### ISSUE TITLE
infected files deleted instead of logged

### ISSUE THREAD

**Issue by himbeere** [nextcloud/files_antivirus#78]
1. Try to upload eicar.com virus
2. Upload is denied
3. File is not on the server

### Expected behaviour
I would expect to get a notification and the file to be on the server. Especially because I set "log only" on settings page.

### Actual behaviour
File is being deleted.

**Operating system**: Ubuntu 16.04

**Web server:** Apache2

**Database:** Maria

**PHP version:** 7.0

**Nextcloud version:** (see Nextcloud admin page) 13.0.0

**Where did you install Nextcloud from:**  tar.gz

#### Nextcloud log (data/owncloud.log)
```
{"reqId":"9P1RDMHBkQOBdzLYc8ZO","level":3,"time":"2018-03-12T10:50:20+00:00","remoteAddr":"","user":"admin","app":"files_antivirus","method":"PUT","url":"\/remote.php\/webdav\/eicar.com","message":"Infected file deleted. Eicar-Test-Signature File: files\/eicar.com.ocTransferId505350298.part Acccount: admin","userAgent":"Mozilla\/5.0 (Macintosh; Intel Mac OS X 10.13; rv:58.0) Gecko\/20100101 Firefox\/58.0","version":"13.0.0.14"}
```

**Comment by rullzer** [comment]
The setting says: `When infected files are found during a background scan`

**Comment by himbeere** [comment]
Ok i see. Sorry then for the noise.

**Comment by lars-sh** [comment]
Actually we ran accross the exactly same problem.
1. Isn't it possible to call the current option "Only Log (on background scans)".
2. And add a third state, e.g. called "Only Log (on background scans and uploads)"?

As far as I figured it out, such a third state needs to be implemented at https://github.com/nextcloud/files_antivirus/blob/41511eccc47d950e35e20daac3174335f1e0475d/lib/AvirWrapper.php#L86 and following only. Am I right?

In case someone is going to approve that, I'd start working.
